### DISCUSSION TITLE
Question about finetuning

### DISCUSSION THREAD

**Discussion by JannikSchneider12** [Noble-Lab/casanovo#299] [Q&A] [answered]
Hello everyone,

I wanted to try casanovo and finetune a model with my data. In the docs I found this part "It is unfortunately not possible to finetune a pre-trained Casanovo model to add new types of PTMs. Instead, such a model must be trained from scratch", but in the tutorial notebook, I found this part: "You can also fine-tune a previously trained model with your own data, specify the starting model weights using --model and set train_from_scratch: False in the config file". Am I right that is in general possible to use a pretrained model and finetune, but just not when including new PTMs ?

Thanks for your time and help

**Comment by wsnoble** [comment, marked as answer]
Yes, that is correct. You can fine-tune the model, but you can't change the alphabet during the fine tuning stage.
